molgenis-security/src/test/java/org/molgenis/security/account/AccountControllerTest.java

```diff
@@ -202,21 +202,6 @@ public void registerUser_activationModeAdmin() throws Exception
 		verify(captchaService).validateCaptcha("validCaptcha");
 	}
 
-	@Test
-	public void registerUser_invalidRegisterRequest() throws Exception
-	{
-		// when(accountService.isSelfRegistrationEnabled()).thenReturn(true);
-		this.mockMvc.perform(post("/account/register").param("username", "admin")
-													  .param("password", "adminpw-invalid")
-													  .param("confirmPassword", "adminpw-invalid")
-													  .param("lastname", "min")
-													  .param("firstname", "ad")
-													  .param("captcha", "validCaptcha")
-													  .contentType(MediaType.APPLICATION_FORM_URLENCODED))
-					.andExpect(status().isBadRequest());
-		verify(captchaService, times(0)).validateCaptcha("validCaptcha");
-	}
-
```

Comment:
I think I've seen it succeed on this branch.

Reply:
I'm quiet sure it failed earlier on. I removed it because I didn't understand what the test was supposed to test and it smelled like a false positive. In the current state of this PR the test is passing again. Not sure why, so reintroduced it.